core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -288,7 +293,12 @@ enum class UniqueType(
     RequiresBuildingInAllCities("Requires a [buildingFilter] in all cities", UniqueTarget.Building),
     @Deprecated("as of 4.10.17", ReplaceWith("Only available <if [buildingFilter] is constructed in at least [positiveAmount] of [All] cities>"))
     RequiresBuildingInSomeCities("Requires a [buildingFilter] in at least [positiveAmount] cities", UniqueTarget.Building),
-    CanOnlyBeBuiltInCertainCities("Can only be built [cityFilter]", UniqueTarget.Building),
+    /** See [OnlyAvailable]. Doesn't restrict Upgrade/Transform pathways
+     */
+    CanOnlyBeBuiltInCertainCities("Can only be built", UniqueTarget.Building, UniqueTarget.Unit,
+        docDescription = "Meant to be used together with conditionals, like \"Can only be built <after adopting [policy]> <while the empire is happy>\". Will also NOT block Upgrade and Transform actions"),
+    @Deprecated("as of x.xx.xx", ReplaceWith("Can only be built <in [cityFilter] cities>"))
+    CanOnlyBeBuiltInCertainCitiesDep("Can only be built [cityFilter]", UniqueTarget.Building),
```

Comment:
Naming nitpick
1. Adding "Dep" to older uniques is pretty rare. Most cases Are when the message of the new unique is almost exactly the same
2. The new unique no longer has a cityFilter, so "InCertainCities" seems a bit off as a name. If we're follow OnlyAvailableWhen as and example, why not "CanOnlyBeBuiltWhen"?

Reply:
Also keep in mind that technically, these two uniques functions differently. The older one didn't block you if it didn't meet the conditionals and the current one does. E.g. "Can only be built [in all cities] <before the [Ancient era]>" will functionally do nothing while "Can only be built <in [all] cities> <before the [Ancient era]>" under this PR will block building the construction at all

Reply:
1. Yes, because the text is the same. But switching [cityFilter] to requires Conditionals. Was trying to match (extend) the original instead of create a new "Only buildable" (which is note a word) to mirror "Only available"
2. Will change name.
3. Good point to notify Modders about. Any conditionals will need to be inverted. Although arguably then the previous implementation doesn't make sense grammatically

Reply:
Actually, going to definitely move next to OnlyAvailable since those are general Construction Uniques instead of the Building-only Uniques.

Reply:
Well, the point previously about location is the automatic unique documentation goes in order from top to bottom with each unique in a category that hasn't been written yet. This rewrites the order of building uniques to be this new one, then cost uniques, then Unbuildable. While it makes sense in the code to be next to OnlyAvailableWhen, it doesn't make make sense due to this quirk of the documentation. Nitpick, though

Reply:
I see. And for whatever reason OnlyAvailable gets pulled to a Tech Unique? Hm.

Why aren't we just duplicating them in the categories? I suppose just makes the page extra long?

Looking at the page seems like Building takes precedent over Unit so this would remain in the Building category.